### ISSUE TITLE
image_classification_model(Large resolution fine-tuning (384x384))

### ISSUE THREAD

**Issue by ElliottZhen** [Sense-X/UniFormer#128]
Hello, is the dim size 64 for the pre-trained model of 384×384 size in image classification?
![1705546116891](https://github.com/Sense-X/UniFormer/assets/56305119/719e0747-ac72-4300-84ff-0aca58bedc0b)

**Comment by Andy1621** [comment]
Hi, for these models, they use `dim=32`, you can find the related model shape in `run.sh`.

https://github.com/Sense-X/UniFormer/blob/52a0415e4e0dda880fab8ffbc710e4d8e171149d/image_classification/token_labeling/exp/uniformer_small_plus_tl_384/run.sh#L4

**Comment by ElliottZhen** [comment]
I see. Thank you very much
